pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
     "ruff>=0.11.1",
     "scikit-learn>=1.6.1",
     "seaborn>=0.13.2",
+    "thefuzz>=0.22.1",
```

Comment:
Do we want this to be part of the default deps?

Reply:
@clefourrier we'd like this to produce a much better validated final dataset - because questions with higher citation scores are more likely to be valid questions compared to ones without